components/dash-core-components/src/components/Textarea.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import {pick} from 'ramda';
+import {PersistedProps, PersistenceTypes, TextAreaProps} from '../types';
+import './css/textarea.css';
+
+const textAreaProps = [
+    'id',
+    'form',
+    'name',
+    'placeholder',
+    'wrap',
+    'accessKey',
+    'contextMenu',
+    'dir',
+    'draggable',
+    'lang',
+    'spellCheck',
+    'style',
+    'title',
+] as const;
+
+const asNumber = (value?: string | number): number | undefined => {
+    return typeof value === 'string'
+        ? isNaN(parseInt(value, 10))
+            ? undefined
+            : parseInt(value, 10)
+        : value;
+};
+const asBool = (value?: string | boolean): boolean | undefined => {
+    if (typeof value === 'string') {
+        if (['true', 'TRUE', 'True'].includes(value)) {
+            return true;
+        }
+        if (['false', 'FALSE', 'False'].includes(value)) {
+            return false;
+        }
+        return undefined;
+    }
+    return value;
+};
```

Comment:
I don't think those conversion are necessary, the props are transfered in json and bool and number. The prop type should be bool or number but the string interpolation makes it harder on the python type check.